content/blog/2022/2022-02-13-nodejs-modules.md

```diff
@@ -0,0 +1,395 @@
+---
+title: "How to Use Modules in Your NodeJS Application?"
+categories: ["NodeJS"]
+date: 2022-02-13 00:00:00 +1100 
+modified: 2022-02-13 00:00:00 +1100
+authors: ["robert"]
```

Comment:
Please add yourself to the file `data/authors.yml` and ideally add a profile picture. It's currently not shown everywhere, but I plan to add an author blurb to each article and create a page for each author with all their articles.